src/middlewared/middlewared/api/v25_10_0/smb.py

```diff
@@ -1,18 +1,25 @@
+from typing import Annotated, Any, Literal, Self
+from pydantic import (
+    AfterValidator, Field, field_validator, IPvAnyInterface, model_validator, ValidatorFunctionWrapHandler
```

Comment:
`ValidatorFunctionWrapHandler` is not used.
```suggestion
    AfterValidator, Field, field_validator, IPvAnyInterface, model_validator
```